src/v2/Apps/IdentityVerification/IdentityVerificationApp/CompleteWatchlist.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Button, Sans, Serif } from "@artsy/palette"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import React from "react"
+import { Title as HeadTitle } from "react-head"
+
+export const CompleteWatchlist: React.FC = () => {
+  return (
+    <AppContainer>
+      <HeadTitle>Artsy | ID Verification</HeadTitle>
+
+      <Box px={[2, 3]} mb={6} mt={4}>
+        <Box mx="auto" width={[335, "80%"]} maxWidth="400px" textAlign="center">
```

Comment:
The nested `<Box>`es could probably be consolidated.

```suggestion
        <Box px={[2, 3]} mb={6} mt={4} mx="auto" width={[335, "80%"]} maxWidth="400px" textAlign="center">
```

Reply:
OK, I copied it from the Processing component.